frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -401,6 +444,45 @@ describe("collection permissions", () => {
               });
             });
 
+            describe("managing question from the question's edit dropdown", () => {
+              it("should not be offered to save question in collections they have `read` access to", () => {
```

Comment:
This is not "saving a question in a collection". We're adding a question to the dashboard that lives in a collection we have 'read' access to. The difference between the two scenarios is huge.

Reply:
My bad. Found and fixed one more inconsistent test name. Thank you!